test/grouping.jl

```diff
@@ -240,4 +240,19 @@ module TestGrouping
             @test size(v) == (2,2)
         end
     end
+
+    @testset "getindex" begin
+        df = DataFrame(a = repeat([1, 2, 3, 4], outer=[2]),
+                       b = repeat([2, 1], outer=[4]))
+        gd = groupby(df, :a)
+        @test gd[1] isa SubDataFrame
```

Comment:
While you're at it, better test that the result is equal to what you expect. Same below.

Reply:
OK. :). I have pushed a commit with some additional related features.